README.md

```diff
@@ -28,18 +28,16 @@ Before, if you were wanting to reproduce some data intensive work done in the li
-    * Aside:  Historically, the data holder (e.g., IRSA) typically covers the cost (e.g., egress) of delivering the data to the user, but cloud storage buckets are starting to change both the workflows and the cost models. Buckets support large-scale data access in ways that an archive like IRSA cannot support from on-premise. This is great, but also means more data requests and larger (and less predictable) costs. Data holders can often get the costs covered through grants, arrangements with the cloud provider (e.g., AWS), etc. But, in some cases they will decide that the best option is to make the data available in a bucket under what I would call the "cloud-pricing model" and sum up as "everyone pays for what *they* use". In particular, this means the data holder will pay the *storage* costs (which they can predict and easily control), and the requestor/end user will pay the *access* costs including egress (which the data holder cannot easily predict or control, but the requestor can). Individual charges are generally small and reasonable when read around in this way, and the the cloud provider often offers free access up to some small but reasonable limit. However, egress is a particular fee that often does not apply at all, but in other cases can balloon to $$$ very quickly. So it is absolutely something to be considered, planned for, and controlled. Lastly, who pays the access/egress costs is determined by a setting on the bucket. If the bucket is "requestor pays", the user/requestor will need (e.g.,) AWS credentials to access it -- charges are then billed to the AWS account that owns the credentials.
-    * As far as I can tell, the best (AWS) option for actual controls -- beyond just "monitoring" support -- is [AWS Budgets Actions](https://aws.amazon.com/blogs/aws-cloud-financial-management/get-started-with-aws-budgets-actions/).
-    * AWS may charge *ingress* fees to bring data into an SMCE pod or user instance. This would be completely separate from any egress fees. There Someone working more directly on Fornax Daskhub would need to answer whether/how ingress applies.
+### How does it impact data access?
+  * Fornax does not restrict data access. Users ought to be able to access any data they both want and have permissions for regardless of where it is (NASA archive, personal computer, AWS S3, Google's GCS, etc.).
+  * Data access through popular tools like astroquery, curl, TAP, etc. should work the same when used on the science console as they do locally.
+  * One difference that may occur is speed of data retrieval, either slower or faster. This can be caused by factors like bandwidth and how far away the data is.
+  * AWS S3 buckets with data curated by the Fornax archives are mounted under `s3/` in the user's `$HOME` directory and are browsable as if they were on the local filesystem.
```

Comment:
I don't see we defined the term "Fornax archives", is this something we will use as an umbrella for the NASA archives, or maybe better to keep spelling them out?

Reply:
At the very bottom of the README it says:

```
Fornax is the collaboration of three NASA archives
- [IRSA](https://irsa.ipac.caltech.edu/frontpage/)
- [HEASARC](https://heasarc.gsfc.nasa.gov/)
- [MAST](https://archive.stsci.edu/)
```

Maybe that should move to the top where it's more visible.